### PR TITLE
#NEAF-76 create new merge method for merging without weather

### PR DESCRIPTION
**Pull request by AlexKostritsa**

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -2,6 +2,10 @@ file {
   inputPath = "src/main/resources/"
   input {
     inputFileNYAccidents = "data.csv"
+    years {
+      first = "2016.csv"
```
Comment (sleshJdev) [inline src/main/resources/application.conf:6]: first/second denote range of years? If so, then I'd rename those params to start/end, from/to or something like that

File: src/main/scala/com/itechart/ny_accidents/service/MergeService.scala [hunk 2]
```diff
@@ -30,16 +30,43 @@ class MergeService @Inject()(weatherService: WeatherMappingService,
...
             data
         }
-      case None => createMergedData(value)
+      case None => fullMergeData(value)
     }
+  }
+
+  def withoutWeatherMapper(value: Accident): MergedData = {
+    if (counter % 100 == 0)
+      println("COUNTER: " + counter)
```
Comment (alexander-belyaev) [inline src/main/scala/com/itechart/ny_accidents/service/MergeService.scala:43]: remove println statements please

**Review by sleshJdev — CHANGES_REQUESTED** [review]

File: src/main/resources/application.conf (hunk 1 above)
Comment (AlexKostritsa) [reply src/main/resources/application.conf:6]: it's recomended oreder, but in code it will always check wich year is erlear

Comment (sleshJdev) [reply src/main/resources/application.conf:6]: Please post a link to code where it happens

Comment (AlexKostritsa) [reply src/main/resources/application.conf:6]: we have discussed it

File: src/main/scala/com/itechart/ny_accidents/service/MergeService.scala (hunk 2 above)
Comment (AlexKostritsa) [reply src/main/scala/com/itechart/ny_accidents/service/MergeService.scala:43]: removed

Comment (AlexKostritsa) [reply src/main/scala/com/itechart/ny_accidents/service/MergeService.scala:43]: I've removed them in NEAF-84, that is a child of this branch

**Pull request closed without merging**